platform/apns/worker.go

```diff
@@ -45,10 +46,13 @@ func (w *Worker) Run(ctx context.Context) error {
 			err = w.updatePushInfoFromTokenUpdate(ctx, event.Message)
 		}
 		if err != nil {
-			level.Info(w.logger).Log(
+			innerErr := level.Info(w.logger).Log(
```

Comment:
logger should not be checked for error. the error is only used for audit logging.
FYI, `fmt.Print` returns an error too. 
https://golang.org/pkg/fmt/#Println

Reply:
Ah, you're not wrong. I'll rip out all the logger error checking.